CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.0.0] - 2023-02-10
+
+### Breaking
+
+- Replaced `shouldOmitPrefix` with `prefixToOmit`. Now an optional string is provided instead of a boolean.
+- This string will be replaced from the start of generated URL path for the given route.
+
```

Comment:
Thought about just having backward compatibility, but it seems needlessly complex; we just have a few places we use this and can update all such occurrences where this would be a breaking change in 1 go. Do we use no-hassle outside of the mono repo?